docs/about-lisk/bridged-tokens.md

```diff
@@ -22,10 +22,10 @@ If you want to add a token to the list, please check out the guide [Bridging an
 
 ## Lisk Mainnet
 
-| Bridged Tokens Mainnet | Symbol | L1 Token          | L2 Token           |
-| :----------------- | :----- |:----------------- | :----------------- |
+| Bridged Tokens Mainnet | Symbol | L1 Token          | L2 Token           | Bridges |
```

Comment:
What's the significance of the `Bridges` column?

Reply:
This PR also solves liskdocs-39, it is related to that